start.sh

```diff
@@ -1,11 +1,11 @@
-#!/user/bin/env bash
+#!/bin/bash
 
 PUID=${PUID:-314}
 PGID=${PGID:-314}
 
 groupmod -o -g "$PGID" titlecardmaker
 usermod -o -u "$PUID" titlecardmaker
 
-find /maker \! \( -uid $(id -u titlecardmaker) -gid $(id -g titlecardmaker) \) -print0 | xargs -0r chown titlecardmaker:titlecardmaker
+chown -R titlecardmaker:titlecardmaker /maker
```

Comment:
functionally, it would search for all files not already owned and chown them, this cuts out all the steps in between for a much cleaner syntax